src/index.css

```
@@ -122,3 +193,13 @@
     @apply bg-background text-foreground;
   }
 }
+
+@import "tailwindcss";
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)